### PR TITLE
Add Tencent TIM

### PR DESCRIPTION
**Pull request by zhuohoudeputao**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/547)

### CODE REVIEW

File: manifests/Tencent/TIM/3.3.0.21315.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Id: Tencent.TIM
+Name: Tencent TIM
```
Comment (hifocus) [inline manifests/Tencent/TIM/3.3.0.21315.yaml:2]: I would suggest to simply use `TIM` here

**Comment by imba-tjd** [comment]
See discussions in https://github.com/microsoft/winget-pkgs/pull/530#discussion_r428445262

**Review by hifocus — CHANGES_REQUESTED** [review]

**Comment by hifocus** [comment]
> See discussions in [#530 (comment)](https://github.com/microsoft/winget-pkgs/pull/530#discussion_r428445262)

I think using the installer's digital signature could be a better idea, though we do have the same outcome:

image

**Comment by imba-tjd** [comment]
https://play.google.com/store/apps/details?id=com.tencent.mobileqq uses `Tencent Technology (Shenzhen) Company Ltd.`
I don't think it needs to be so rigorous. Anyway I will correspond with you.
As for the blank line, I prefer having. It's *nix style.

**Comment by kidonng** [comment]
#279?

**Comment by imba-tjd** [comment]
:joy: Add @YadominJinta in discussion. I don't know what to do. You and `zhuohoudeputao` decide.

**Comment by YadominJinta** [comment]
I don't think it needs to be so rigorous. 🤣 
But I will follow you suggestions

**Comment by imba-tjd** [comment]
One thing is that both YadominJinta and zhuohoudeputao are trying to add TIM. In theory we should keep the early one.

**Comment by YadominJinta** [comment]
But my pull stuck in CI, I think it's maybe QQ or Tim's problem

**Comment by hifocus** [comment]
> But my pull stuck in CI, I think it's maybe QQ or Tim's problem

Same as mine. I tried to add QQ.

**Comment by zhuohoudeputao** [comment]
Does it seem that it will be stuck here forever?

**Pull request closed without merging**